pre-processing/primod/driver_coupling/ribamod.py

```diff
@@ -106,6 +106,27 @@ def write_exchanges(
                     save_flows=True,
                 )
                 destination = "river"
+
+                #  check on the bottom elevation and ribasim minimal subgrid level
+                minsglevel = (
+                    ribasim_model.basin.subgrid.df.groupby("subgrid_id")
+                    .min()[["subgrid_level"]]
+                    .to_numpy()
+                    .ravel()
+                )
+                # in active coupling, check subgrid levels versus modflow bottom elevation
+                if isinstance(self, RibaModActiveDriverCoupling):
+                    sugindex = mapping.dataframe["subgrid_index"]
+                    bndindex = mapping.dataframe["bound_index"]
+                    modflow_minlevel = package["bottom_elevation"].to_numpy().ravel()
```

Comment:
please use explicit names like minimal_subgrid_level,  subgrid_index and river_index to make the code better readable.

Reply:
changed names to more explicit, removed redundant operations and variables